test/integration/goldens/asset/BUILD.bazel

```diff
@@ -0,0 +1,6 @@
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
+    name = "asset_goldens_files",
```

Comment:
Just `goldens_files` should be sufficient, since this is in the `asset` directory.

Reply:
SG!